src/main/java/com/es/phoneshop/model/BaseDao.java

```diff
@@ -0,0 +1,8 @@
+package com.es.phoneshop.model;
+
+public interface BaseDao<T> {
```

Comment:
Should BaseDao have implementation of getEntity and save methods so they can be used in product, order daos?

Reply:
It was one of the homework tasks